### DISCUSSION TITLE
Some drawings

### DISCUSSION THREAD

**Discussion by FlightBlaze** [DiligentGraphics/DiligentEngine#209] [Ideas]
I drew a mascot in a triangle T-shirt that paints triangles (that's basically what Diligent Engine does), a new logo that consists of triangles, and several icons.

<img width="1440" alt="Screenshot 2022-09-16 at 17 40 07" src="https://user-images.githubusercontent.com/18074076/190665615-119a6dd1-e726-4142-8229-6b7f9e86d474.png">

Hope you like it. Кстати, я тоже русский! Спасибо за такой чудесный движок!

**Comment by FlightBlaze** [comment]
I updated the logo. The colors now more saturated.

If you like it, feel free to use it. If not, I can make another logo or illustration (for free. I just want to be useful).

**Reply by TheMostDiligent** [reply]
Ok, thanks!
